webapi/tct-canvas-html5-tests/html5-canvas-app/data/x86-memo/canvas_colorsandstyles_createlineargradient_001_baseline.md5

```diff
@@ -0,0 +1 @@
+5fdc658cc7b9f789241cc98e69bb42d8
```

Comment:
I think it is better that upload the baseline data after all the bdd tests are completed in webapi.